lib/utils.js

```diff
@@ -95,15 +139,33 @@ const utils = {
         utils.logError('Configuration file missing for this directory, run losant configure to generate this file.');
         return;
       }
-      config.file = file;
+      if (!config.apiUrl) {
+        const url = utils.getApiURL(userConfig);
+        if (url) {
```

Comment:
is it possible to not have a url returned by getApiURL?

Reply:
No, because if loadUserConfig was empty then it would have thrown an error and exited already. loadApplicationConfig is only used in loadConfig after loading the user config.

Reply:
so then you can drop the if statement?